test/crawler_web/controllers/user_auth_controller_test.exs

```diff
@@ -0,0 +1,155 @@
+defmodule CrawlerWeb.UserAuthTest do
```

Comment:
Let remove the `_controller_` in `test/crawler_web/controllers/user_auth_controller_test.exs`